### ISSUE TITLE
Cannot update dataset with new value of field termsOfUse

### ISSUE THREAD

**Issue by didaRo** [IQSS/dataverse#3813]
Hello,

I use dataverse 4.5.1. when I try to update a Dataset DRAFT for the first one with setting new value "termsOfUse", with Native API Dataverse (File json), 
`PUT http://$SERVER/api/datasets/$id/versions/:draft?key=$apiKey`, the Dataset not take the new value.

But when I publish the dataset and after I try second time to update dataset it's work, I have "termsOfUse" with the new value

Thanks!
Fairouz

**Comment by djbrooke** [comment]
Hey Fairouz - is this a duplicate of #3777 ? There's a workaround there that may help you. If it's a duplicate, I'll close this out and we'll plan to just have the one ticket (#3777).

**Comment by didaRo** [comment]
Hey @djbrooke,
It is not same problem. Native API not taken metadata "termsOfUse" value if dataset is not published

Scenario with Native API is : 
Case 1
- Add a new dataset  with metadata "termsOfUse"
- Publish the dataset 
- update metadata  "termsOfUse" of dataset
All it's OK

Case2
- Add a new dataset  with metadata "termsOfUse"
- update metadata  "termsOfUse" of dataset
It's KO I have not the new value of "termsOfUse"

Thanks!
dida

**Comment by djbrooke** [comment]
Thanks @didaRo, one of our developers will have to take a look a this one. Is there an error that's given, or is the field just not updated? Thanks for your patience!

**Comment by didaRo** [comment]
Hey @djbrooke,
There are no error, just field not updated
thanks

**Comment by landreev** [comment]
This appears to be a legitimate issue. 
Unfortunately, more issues were introduced in that API between 4.5.1 and the current version, 4.6.1. That's the issue #3777, that @djbrooke mentioned. So we need to address those new issues as well, before we can fix this one. 
But hopefully we'll be able to address this soon.

**Comment by landreev** [comment]
@djbrooke It's not a duplicate exactly; but definitely depends on #3777. 
In the current build, you can't get even this far.

**Comment by pdurbin** [comment]
#3777 has been closed with pull request #4566 and the new write up is at http://guides.dataverse.org/en/4.9.1/api/native-api.html#update-metadata-for-a-dataset

@didaRo do you feel like re-testing this? Also, this sample JSON file has termsOfUse if you want to try it: https://github.com/IQSS/dataverse/blob/v4.9.1/scripts/api/data/dataset-create-new-all-default-fields.json

**Comment by pdurbin** [comment]
@didaRo #5899 was closed by PR #7414 and I think the new semantic API will help. Can you please give it a try? Please see https://guides.dataverse.org/en/5.11.1/developers/dataset-semantic-metadata-api.html
